v3/src/components/graph/adornments/plotted-function/plotted-function-adornment-model.ts

```diff
@@ -1,22 +1,16 @@
 import { Instance, types } from "mobx-state-tree"
 import { AdornmentModel, IAdornmentModel, IUpdateCategoriesOptions } from "../adornment-models"
-import { kPlottedFunctionType, kPlottedFunctionValueTitleKey } from "./plotted-function-adornment-types"
-import { ScaleNumericBaseType } from "../../../axis/axis-types"
-import { Point } from "../../../data-display/data-display-types"
-import { ICase } from "../../../../models/data/data-set-types"
-import { IGraphDataConfigurationModel } from "../../models/graph-data-configuration-model"
+import { kPlottedFunctionType, kPlottedFunctionValueTitleKey, FormulaFn } from "./plotted-function-adornment-types"
 
 export const MeasureInstance = types.model("MeasureInstance", {})
```

Comment:
Not directly part of this PR, but this `MeasureInstance` and the one in `univariate-measure-adornment-model.ts` should have unique names.

Reply:
Maybe it'd leave it up to @emcelroy, as he knows the differences better and probably can come up with some better naming that I would.

BTW, what's the purpose of this name property? The docs don't explain anything about it. Is it just for debugging?

Reply:
> BTW, what's the purpose of this name property? The docs don't explain anything about it. Is it just for debugging?

It's _mostly_ just for debugging/human readability, but the name is retained at runtime and so it can be used for lookup purposes. In `mst-utils.ts` we have a `getParentWithTypeName()` function which can be used, for instance, to walk up the tree to find the `DocumentContent` model.